.github/workflows/Android-CI.yml

```diff
@@ -5,7 +5,7 @@ on: [pull_request]
 jobs:
   buildTest:
     name: Build & Test JDK ${{ matrix.java_version }}
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
```

Comment:
Whats the reason for the OS change?

Reply:
macOS comes with hardware acceleration to run Emulator with x86 fast

Reply:
Where'd you see that? I didn't see anything mentioned in the [docs](https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners). Cool if it actually runs faster though!

Reply:
I can't tell you exactly, I simply know it. You can test it and change to `ubuntuLatest`, then you see the result